src/org/zaproxy/zap/extension/fuzz/payloads/ui/impl/ScriptStringPayloadGeneratorAdapterUIHandler.java

```diff
@@ -224,6 +360,10 @@ public boolean validate() {
                                 Constant.messages.getString("fuzz.payloads.generator.script.warnNoInterface.message"),
                                 Constant.messages.getString("fuzz.payloads.generator.script.warnNoInterface.title"),
                                 JOptionPane.INFORMATION_MESSAGE);
+                        handleScriptExceptionImpl(
+                                scriptWrapper,
+                                Constant.messages.getString("fuzz.payloads.generator.script.warnNoInterface.message"));
+                        getPayloadsPreviewGenerateButton().setEnabled(false);
                         return false;
                     }
                 } catch (Exception e) {
```

Comment:
In this case it should just be ScriptException and IOException but, script engine implementations might throw different (internal) exceptions.
In some classes [1][2][3] I did add a comment pointing that out, but didn't want to spam those comments over all the code.

[1] https://github.com/zaproxy/zap-extensions/blob/08b9245d97737b474afd971eba433d3ae7dc8e9f/src/org/zaproxy/zap/extension/fuzz/httpfuzzer/processors/FuzzerHttpMessageScriptProcessorAdapter.java#L73-L77
[2] https://github.com/zaproxy/zap-extensions/blob/08b9245d97737b474afd971eba433d3ae7dc8e9f/src/org/zaproxy/zap/extension/fuzz/payloads/processor/ScriptStringPayloadProcessorAdapter.java#L69
[3] https://github.com/zaproxy/zap-extensions/blob/55c21a456cdf606188ebc91a4dfc52e471873635/src/org/zaproxy/zap/extension/fuzz/payloads/generator/ScriptStringPayloadGeneratorAdapter.java#L145